.github/workflows/release_centos8.yaml

```diff
@@ -40,9 +40,13 @@ jobs:
           # Initialize dependencies.
           ./init
 
+          # Set default build_type to Release
+          INPUT_BUILD_TYPE=${{ github.event.inputs.build_type }}
+          BUILD_TYPE=${INPUT_BUILD_TYPE:-"Release"}
```

Comment:
Default build type is set in inputs. What is the purpose of checking again here?

Reply:
The input variable in gh actions has a caveat, the deafult only works when using wokrflow_dispatch, in other words when running the workflow manually. When the workflow is run automatically the entire workflow_dispatch section is skipped and no variable is infered, not even the default

Reply:
So basically this check is redundant only when running manually, but is crucial when running automatically.

Reply:
Thank you for explanation! 🙂